.github/workflows/kaf.yaml

```
@@ -0,0 +1,54 @@
+# kaf.yaml - builds and pushes a Kaf container image.
+#
+# Requires repository variables:
+# - REGISTRY_SERVER - the server of the container registry service e.g. `quay.io` or `docker.io`
+# - REGISTRY_USERNAME - - your username on the service (or username of your robot account)
+# - REGISTRY_TOKEN - the access token that corresponds to `REGISTRY_USERNAME`
+#
+# If the required repository variables aren't set the workflow will be skipped.  This means the workflow won't fail
+# on the forks of developers who haven't configured the variables/secrets.
+
+name: Kaf Container 
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'Repository containing kaf'
+        default: 'birdayz/kaf'
+        required: true
+      registry-destination:
+        description: 'The release version, e.g. quay.io/<my org>/kaf'
+        default: 'quay.io/k_wall/kaf'
+        required: true
+      ref:
+        description: 'Ref (branch/tag)'
+        required: true
+        default: 'master'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: ${{ vars.REGISTRY_SERVER != '' && vars.REGISTRY_USERNAME != '' }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository:  ${{ github.event.inputs.repository }}
+          ref: ${{ github.event.inputs.ref }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ vars.REGISTRY_SERVER }}
+          username: ${{ vars.REGISTRY_USERNAME }}
+          password: ${{ secrets.REGISTRY_TOKEN }}
+      - name: Build and push Kaf container image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
+          push: true
+          tags: ${{ github.event.inputs.registry-destination }}:${{ github.event.inputs.ref }}
```

can we try 
```suggestion
          tags: ${{ github.event.inputs.registry-destination }}:${{ github.event.inputs.ref }}
          cache-from: type=gha
          cache-to: type=gha,mode=max,compression=zstd
```

To reduce build times and thus action minutes consumption?